### DISCUSSION TITLE
Not able to import from the "classnames" package

### DISCUSSION THREAD

**Discussion by gnapse** [fwouts/previewjs#42] [Q&A] [answered]
I'm trying React Preview with [this component](https://github.com/Doist/reactist/blob/beta/src/new-components/text-field/text-field.tsx), and I'm getting this error:

```
The requested module '/preview/node_modules/classnames/index.js' does not provide an export named 'default'
```

This library internally uses [`classnames`](https://www.npmjs.com/package/classnames), and that error message seems to be related to it. Would appreciate any pointers on what's going on here. Happy to give more details if needed.

**Comment by fwouts** [comment, marked as answer]
Hey @gnapse, long-time power user of Todoist here, so thank you for your work :)

I think I found the issue. The following TypeScript alias is confusing React Preview (more specifically `vite-tsconfig-paths`, which is used under the hood):
```json
        "paths": {
            "@": ["./"],
            "*": ["src/*", "node_modules/*"]  // this line is the issue (specifically the node_modules alias)
        },
```

The codebase seems to compile fine without it, so maybe you could simply remove it?

Note: You may need to reload your IDE to force React Preview to restart (I'll make sure it restarts automatically when tsconfig.json is updated in the future).

**Reply by gnapse** [reply]
Indeed, that solves the issue. I'll figure out if removing that has some negative implication, but it seems we could get rid of it. Thanks!

> Hey @gnapse, long-time power user of Todoist here, so thank you for your work :)

Thank you! I'll make sure to pass the appreciation along to the rest of us at Doist.
